src/scripts/angry-tock.js

```diff
@@ -112,64 +117,61 @@ const shout = async ({ calm = false } = {}) => {
  */
 const isAngryTockDay = (now) => {
   const d = now || m();
-  return d.format("dddd") === "Monday" && !holidays.isAHoliday(d.toDate());
+  return d.dayOfWeek === 1 && !holidays.isAHoliday(dateFromZonedDateTime(m()));
 };
 
 /**
  * Schedules the next time to shout at users.
  */
 const scheduleNextShoutingMatch = () => {
-  const day = moment.tz(ANGRY_TOCK_TIMEZONE);
-
-  const firstHour = ANGRY_TOCK_FIRST_ALERT.hour();
-  const firstMinute = ANGRY_TOCK_FIRST_ALERT.minute();
-  const firstTockShoutTime = day.clone();
-  firstTockShoutTime.hour(firstHour);
-  firstTockShoutTime.minute(firstMinute);
-  firstTockShoutTime.second(0);
-
-  const secondHour = ANGRY_TOCK_SECOND_ALERT.hour();
-  const secondMinute = ANGRY_TOCK_SECOND_ALERT.minute();
-  const secondTockShoutTime = day.clone();
-  secondTockShoutTime.hour(secondHour);
-  secondTockShoutTime.minute(secondMinute);
-  secondTockShoutTime.second(0);
+  const day = m();
+
+  const firstTockShoutTime = day.withPlainTime(ANGRY_TOCK_FIRST_ALERT);
+  const secondTockShoutTime = day.withPlainTime(ANGRY_TOCK_SECOND_ALERT);
+
+  let next = day;
 
   if (isAngryTockDay(day)) {
     // If today is the normal day for Angry Tock to shout...
-    if (day.isBefore(firstTockShoutTime)) {
+    if (Temporal.ZonedDateTime.compare(day, firstTockShoutTime) < 0) {
+      // day.isBefore(firstTockShoutTime)) {
```

Comment:
I know this is in draft, but this commented out line will go away right?

Reply:
Ooh, good catch. Thanks!